app/user/permissions.py

```diff
@@ -0,0 +1,12 @@
+from typing import Any
+
+from rest_framework import permissions
+from rest_framework.request import Request
+from rest_framework.views import APIView
+
+
+class IsUser(permissions.BasePermission):
+    def has_object_permission(
+        self, request: Request, view: APIView, obj: Any
+    ) -> Any:
+        return obj.user == request.user
```

Comment:
what if the admin needs to edit?, eg incase of inappropriate bio or image

Reply:
He is able to edit from the admin panel. As long as he's logged in.